src/components/NewClusterModal.js

```diff
@@ -187,9 +187,10 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
 
     const makeEnvSelect = id => h(Select, {
       id,
+      'aria-label': 'Select Environment',
       value: selectedLeoImage,
       onChange: ({ value }) => this.setState({ selectedLeoImage: value }),
-      isSearchable: false,
+      isSearchable: true,
```

Comment:
TOL: If this improves a11y, then I guess this is fine. Otherwise, I don't understand why we need to make this change in order to be able to drive the UI with puppeteer. Inspecting the select options is difficult in the browser so I don't have alternative suggestions without spending more time with it.